docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,171 @@
+- [How to: customize a time-based cache policy](../docs/framework/network-programming/how-to-customize-a-time-based-cache-policy.md)
+- [Using TCP Services](../docs/framework/network-programming/using-tcp-services.md)
+- [Application Domain Resource Monitoring (ARM) ETW Events](../docs/framework/performance/application-domain-resource-monitoring-arm-etw-events.md)
+- [Garbage Collection ETW Events](../docs/framework/performance/garbage-collection-etw-events.md)
+- [Creating Satellite Assemblies for Desktop Apps](../docs/framework/resources/creating-satellite-assemblies-for-desktop-apps.md)
+- [.NET Framework Tools](../docs/framework/tools/index.md)
+- [How to: Import Custom WSDL](../docs/framework/wcf/extending/how-to-import-custom-wsdl.md)
+- [How to: Write an Extension for the ServiceContractGenerator](../docs/framework/wcf/extending/how-to-write-an-extension-for-the-servicecontractgenerator.md)
+- [Flowing Transactions into and out of Workflow Services](../docs/framework/wcf/feature-details/flowing-transactions-into-and-out-of-workflow-services.md)
+- [Custom WSDL Publication](../docs/framework/wcf/samples/custom-wsdl-publication.md)
+- [Extending Control Over Error Handling and Reporting](../docs/framework/wcf/samples/extending-control-over-error-handling-and-reporting.md)
+- [How to create and run a long-running workflow](../docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md)
+- [About GDI+ Managed Code](../docs/framework/winforms/advanced/about-gdi-managed-code.md)
+- [How to: Read Image Metadata](../docs/framework/winforms/advanced/how-to-read-image-metadata.md)
+- [Threading Model](../docs/framework/wpf/advanced/threading-model.md)
+- [Navigation Overview](../docs/framework/wpf/app-development/navigation-overview.md)
+- [Adorners Overview](../docs/framework/wpf/controls/adorners-overview.md)
+- [Data Templating Overview](../docs/framework/wpf/data/data-templating-overview.md)
+- [How to: Control When the TextBox Text Updates the Source](../docs/framework/wpf/data/how-to-control-when-the-textbox-text-updates-the-source.md)
+
+## F# language
+
+- [Get Started with F# in Visual Studio Code](../docs/fsharp/get-started/get-started-vscode.md)
+- [https://raw.githubusercontent.com/dotnet/docs/master/docs/fsharp/index.yml](../docs/fsharp/index.yml)
+- [Computation Expressions](../docs/fsharp/language-reference/computation-expressions.md)
+- [Sequences](../docs/fsharp/language-reference/sequences.md)
+- [F# code formatting guidelines](../docs/fsharp/style-guide/formatting.md)
+- [What's new in F# 4.5 - F# Guide](../docs/fsharp/whats-new/fsharp-45.md)
+- [What's new in F# 4.6 - F# Guide](../docs/fsharp/whats-new/fsharp-46.md)
+- [What's new in F# 4.7 - F# Guide](../docs/fsharp/whats-new/fsharp-47.md)
+
+## Contributors
+
+We'd like to recognize the following people, who've contributed to.NET docs since our last release notes:
+
+- [Jeej](https://github.com/Jeej). 
```

Comment:
Would it be feasible to order by number of contributions, most to least? If not, I suggest we put those without names at the end of the list instead of the beginning.

Reply:
I could sort by number of contributions. I'll write that code.

Reply:
Agreed with @tdykstra on the ordering. Maybe add a link to how to contribute to docs and be on that list? 🤔